src/Workspaces/VisualBasic/Portable/LanguageServices/VisualBasicSyntaxFactsService.vb

```diff
@@ -19,6 +19,8 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         Inherits AbstractSyntaxFactsService
         Implements ISyntaxFactsService
 
+        Public Shared ReadOnly Property [Default] As New VisualBasicSyntaxFactsService
```

Comment:
Sure - but that's because something about those services requires the `languageServices`.  In this case, nothing in `VisualBasicSyntaxFactsServices` needs it.